src/entities/Client.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, Column, BeforeInsert, BeforeUpdate, CreateDateColumn, UpdateDateColumn, JoinColumn, ManyToOne, PrimaryColumn, BaseEntity} from "typeorm";
+import * as bcrypt from "bcrypt";
+import { Role } from "./Role";
+@Entity('users')
+export class Client extends BaseEntity {
```

Comment:
why the entity name is different from the class name ?

Reply:
I will change it sorry for the typo :(